src/pp-annotator/api/DebugStorage.ts

```diff
@@ -14,6 +17,11 @@ export default class DebugStorage implements IStorage, IModule {
 
   trace(action: string, data: any) {
     console.debug(`DebugStorage: invoking action ${action} on data: `, data);
+    // If some annotations are hardcoded, print an annotation json to facilitate copying it manually to the file
+    if (PP_SETTINGS.READ_ANNOTATIONS_FROM_FILE) {
+      console.debug('Annotation json:');
+      console.debug(JSON.stringify(data));
```

Comment:
Now every time something is done (and the env var is set), this JSON is printed. Is this intentional?

Reply:
Yes (I should've made a comment in the PR description)

As a substitute to a database, to create a persistent annotation in the application (for demo purposes), you can copy the json from the console output, paste it into `annotations.json` and there you are ;)